package.json

```diff
@@ -64,6 +64,7 @@
   "dependencies": {
     "@webcomponents/shadycss": "^1",
     "@webcomponents/webcomponentsjs": "^2",
+    "d2l-inputs": "BrightspaceUI/inputs#semver:^2",
     "d2l-intl": "^2",
     "fastdom": "^1.0.9",
```

Comment:
Interesting that `fastdom` is here.  I think this branch might be out of date.

Reply:
fastdom was added in the previous PR for the change in breakpoints.

Reply:
I think we're really hoping to avoid using it in Lit. It shouldn't be necessary.

Reply:
You are right! we shouldn't need it. I'll update PR 1 to stop using it. Still getting used to all the new life cycle controls we have but those are what I should be using.